co2calculator/distances.py

```diff
@@ -142,6 +143,32 @@ def geocoding_airport(iata: str) -> Tuple[str, Tuple[float, float], str]:
     return name, geom, country
 
 
+def geocoding_airport(iata: str) -> Tuple[str, Tuple[float, float], str]:
+    """Function to obtain the coordinates of an airport by the IATA code
+
+    :param iata: IATA airport code
+    :type iata: str
+    :return: name, coordinates and country of the found airport
+    :rtype: Tuple[str, Tuple[float, float], str]
+    """
+    global df_airports
+    if df_airports is None:
+        df_airports = pd.read_csv(
```

Comment:
really cool that it can be read from url! Since the data is downloaded each time the function is called, maybe it's better to create the dataframe only once when the module is loaded, so in line 32?